### DISCUSSION TITLE
🐛 labels配置selector不生效

### DISCUSSION THREAD

**Discussion by yuweiwen007** [antvis/G2#5612] [General]
![image](https://github.com/antvis/G2/assets/79632701/98cab79f-2051-476c-903d-70dafaec30e7)
![image](https://github.com/antvis/G2/assets/79632701/9d07273f-7e42-4e14-8d24-2061900a03da)

**Comment by yuweiwen007** [comment]
type: 'interval'

**Reply by pearmini** [reply]
这个有完整的案例吗？

**Comment by yuweiwen007** [comment]
另外，selector为函数时，console.log({ data }) 为 数组的每一项，而非整个数组
![image](https://github.com/antvis/G2/assets/79632701/d7ad4821-587e-42a8-b6c8-8e7e54639002)
